.circleci/config.yml

```diff
@@ -117,13 +142,16 @@ workflows:
   version: 2
   lint-buildandtest-compileoutput-checkdeploy:
     jobs:
-      - build-and-test
       - lint
+      - spm-build-and-test
+      - xcode-build-and-test
       - compile-output:
           requires:
-            - build-and-test
             - lint
+            - spm-build-and-test
```

Comment:
Not really related to that PR specifically, but I wonder why `compile-output` requires those tasks to pass before `swiftc`-ing the Tests/Fixtures/Generated outputs (which are static fixtures)?

Reply:
That's something I've done for both this repository and StencilSwiftKit. I've moved all longer  (and less important) steps in such a way that they'll only execute if the other jobs succeed. This is to have quicker results 😄 

This boils down to moving the CocoaPods spec check to later in the flow, both because it has to download the full spec repo, but also because it's something that's less likely to change during PRs. Same thing for the `compile-output`. The `lint` step isn't that important (priority-wise), but it's very quick to execute.

Reply:
Ok makes sense, and that's actually what I thought the reason was 😅